Form-Controls/index.html

```diff
@@ -1,28 +1,105 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Product Pick</h1>
+    </header>
+    <main>
+      <form>
+        <fieldset class="details1">
+          <legend>choose your favorite T-shirt</legend>
+          <div>
+            <label for="black"
+              ><img
+                class="color_img"
+                alt="Black T-shirt"
+                src="https://static.thcdn.com/productimg/1600/1600/13254031-1024901106882108.jpg"
+              /><br />
```

Comment:
I know you are trying to format it for readability. But you should try keep certain tags like <img> into 1 line if you can. Long as it is readable. Ultimately, the more lines you have the longer it will take browser to load the files so you want to minimise things for performance. Hope this make sense :)

Reply:
first of all thank you to review my codes,  i was looking why my performance is not 100% i didn't figure out is because of that. I will use local path , but about line when i save my code in Vs code ,it is just changed  in this way. I will look to find out why?